app/src/main/java/com/example/movie_list/ui/components/MovieDetailComponent.kt

```diff
@@ -0,0 +1,79 @@
+package com.example.movie_list.ui.components
+
+import android.content.Context
+import android.widget.LinearLayout
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.CenterCrop
+import com.example.movie_list.actions.ActionCreator
+import com.example.movie_list.model.Movie
+import trikita.anvil.Anvil
+import trikita.anvil.BaseDSL
+import trikita.anvil.DSL.*
+import trikita.anvil.RenderableView
+
+inline fun movieDetailComponent(crossinline func: MovieDetailComponent.() -> Unit) {
+    highOrderComponent(func)
+}
+
+class MovieDetailComponent(context: Context) : RenderableView(context) {
+
+    var movieTitle: String = "" // Is that correct?
+    var movieOverview: String = ""
+    var movieDate: String = ""
+    var movieBackdrop: String = ""
+    var movieList: List<Movie> = emptyList()
+    val BACKDROPID = 100
+    val TITLEID = 102
+    val DATEID = 103
+    val OVERVIEWID = 104

```

Comment:
This is the way that constants are declared on Kotlin
```suggestion
    companion object {
      const  val BACKDROPID = 100
      const val TITLEID = 102
      const val DATEID = 103
      const val OVERVIEWID = 104
    }
```